docs/external_api/file_manager.md

```diff
@@ -273,10 +272,15 @@ GET /server/files/metadata?filename=tools/drill.gcode
 | `first_layer_extr_temp` |  float   | The configured first layer extruder temperature, in Celsius. |
 | `first_layer_bed_temp`  |  float   | The configured first layer bed temperature, in Celsius.      |
 | `chamber_temp`          |  float   | The configured chamber temperature, in Celsius.              |
-| `filament_name`         |   str    | The name of the filament used.                               |
-| `filament_type`         |   str    | The type of filament used, ie: `PLA`.                        |
+| `filament_name`         |   str    | The name(s) of the filament used.                            |
+| `filament_color`        |   str    | The colors(s) of the filament used in #RRGGBB format.        |
+| `extruder_color`        |   str    | The slicer defined extruder color(s) for the print.          |
+| `filament_temp`         |   str    | The base temperature(s) of filament used, in Celsius.        |
```

Comment:
I noticed that this is parsed as a string.  Other temperatures parsed by metadata.py are expressed as a float, is there a particuar reason for this not to be?  Is it because there may be multiple?  Since this is a new addition it could be returned as a list of floats.  I'm open to discussion on which is best.